scripts/update/rtorrent.sh

```diff
@@ -1,45 +1,59 @@
 #!/bin/bash
-users=($(cut -d: -f1 < /etc/htpasswd))
 
-for u in "${users[@]}"; do
-    if [[ ! -f /etc/tmpfiles.d/${u}.conf ]]; then
```

Comment:
This is a tough one because the tmpfiles for the user exists with or without rTorrent being installed. It's generated during user creation whether or not rTorrent is installed.